benchmark/tpcc-v2.md

```diff
@@ -0,0 +1,111 @@
+---
+title: TiDB TPC-C Performance Test Report -- v4.0 vs. v3.0
+category: benchmark
```

Comment:
Please add a summary in this metadata section.

Reply:
Updated. Thanks!